lib/index.js

```diff
@@ -1,3 +1,30 @@
 const React = require('react');
 const ReactDOM = require('react-dom');
 
+
+class HarvestHelper extends React.Component {
+  constructor() {
+    super();
+    this.state = {};
```

Comment:
I am going to eventually save things on the state and am working on that now.  You do not need to set it if you have nothing you're storing there.